tutorials/integration.md

```diff
@@ -488,34 +543,40 @@ const flagAbi = [
   ];    // flagAbi
 ```
 
+&nbsp;
+
 This function calls the contract using GSN. It is standard `ethers.js`, provided here for 
 completeness.
 ```javascript
 const gsnContractCall = async () => {
-
-	await window.ethereum.enable();
-
+	await startGsn()
```

Comment:
Why is `startGsn` here? First, it is confusing as we have `GsnTestEnvironment.startGsn` that literally starts the GSN on a local computer, which is not what this 'startGSN' does. 
Also, this code will only initialize the GSN-related variables once the user clicks or whatever, which is a weird timing. And will be called multiple times instead of once.


As the purpose of the tutorial is to showcase the library, the code must be dogmatic, trivial and immediately understandable. This does not feel like it.

Reply:
Renamed